diff -Naur vendor/github.com/containers/storage/drivers/copy/copy.go vendor2/github.com/containers/storage/drivers/copy/copy.go
--- vendor/github.com/containers/storage/drivers/copy/copy.go	2019-03-26 10:45:04.518725429 -0400
+++ vendor2/github.com/containers/storage/drivers/copy/copy.go	2019-03-26 10:48:33.176062993 -0400
@@ -1,4 +1,4 @@
-// +build linux
+// +build cgo
 
 package copy
 
diff -Naur vendor/github.com/containers/storage/drivers/copy/copy_unsupported.go vendor2/github.com/containers/storage/drivers/copy/copy_unsupported.go
--- vendor/github.com/containers/storage/drivers/copy/copy_unsupported.go	1969-12-31 19:00:00.000000000 -0500
+++ vendor2/github.com/containers/storage/drivers/copy/copy_unsupported.go	2019-03-26 10:28:19.815296691 -0400
@@ -0,0 +1,19 @@
+// +build !linux !cgo
+
+package copy
+
+import "github.com/containers/storage/pkg/chrootarchive"
+
+// Mode indicates whether to use hardlink or copy content
+type Mode int
+
+const (
+	// Content creates a new file, and copies the content of the file
+	Content Mode = iota
+)
+
+// DirCopy copies or hardlinks the contents of one directory to another,
+// properly handling soft links
+func DirCopy(srcDir, dstDir string, _ Mode, _ bool) error {
+	return chrootarchive.NewArchiver(nil).CopyWithTar(srcDir, dstDir)
+}
